docs/deployment/configuration/auth_setup.rst

```diff
@@ -351,28 +351,27 @@ Apply OIDC Configuration
            # Use the non-encoded version of the random password 
              clientSecret: "<your-random-password>" 
              clientId: flytepropeller
-         
-      8. Save and exit your editor.
 
-      9. Restart ``flyteadmin`` for the changes to take effect:
+.. note::
 
-      .. prompt:: bash $
+   For `multi-cluster deployments <https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html>`__, this secret definition has to be added to the `values-dataplane.yaml` file.
```

Comment:
```suggestion
   For `multi-cluster deployments <https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html>`__ running `flytepropeller` in the control plane cluster, you must add this secret definition to the `values-dataplane.yaml` file.
```

Reply:
I'm trying not to get lost in grammar :) The idea is: in every cluster where propeller is running you need that secret (not only environments where propeller is running on the control plane cluster).
Propeller in the control plane cluster is optional, for the data plane is a must.

Let me know if it's clear or not

Reply:
Gotcha, then I would suggest:
```
For `multi-cluster deployments <https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html>`__, you must add this secret definition to the `values-dataplane.yaml` file.
```